simde/x86/avx2.h

```diff
@@ -1569,31 +1569,15 @@ simde_mm256_hadd_epi16 (simde__m256i a, simde__m256i b) {
   return _mm256_hadd_epi16(a, b);
 #else
   simde__m256i_private r_;
-  simde__m256i_private a_ = simde__m256i_to_private(a);
-  simde__m256i_private b_ = simde__m256i_to_private(b);
 
-  r_.i16[0] = a_.i16[1] + a_.i16[0];
-  r_.i16[1] = a_.i16[3] + a_.i16[2];
-  r_.i16[2] = a_.i16[5] + a_.i16[4];
-  r_.i16[3] = a_.i16[7] + a_.i16[6];
-  r_.i16[4] = b_.i16[1] + b_.i16[0];
-  r_.i16[5] = b_.i16[3] + b_.i16[2];
-  r_.i16[6] = b_.i16[5] + b_.i16[4];
-  r_.i16[7] = b_.i16[7] + b_.i16[6];
-  r_.i16[8] = a_.i16[9] + a_.i16[8];
-  r_.i16[9] = a_.i16[11] + a_.i16[10];
-  r_.i16[10] = a_.i16[13] + a_.i16[12];
-  r_.i16[11] = a_.i16[15] + a_.i16[14];
-  r_.i16[12] = b_.i16[9] + b_.i16[8];
-  r_.i16[13] = b_.i16[11] + b_.i16[10];
-  r_.i16[14] = b_.i16[13] + b_.i16[12];
-  r_.i16[15] = b_.i16[15] + b_.i16[14];
+  r_.m128i[0] = simde_mm_add_epi16(simde_x_mm_deinterleaveeven_epi16(simde_mm256_extracti128_si256(a,0),simde_mm256_extracti128_si256(b,0)),simde_x_mm_deinterleaveodd_epi16(simde_mm256_extracti128_si256(a,0),simde_mm256_extracti128_si256(b,0)));
+  r_.m128i[1] = simde_mm_add_epi16(simde_x_mm_deinterleaveeven_epi16(simde_mm256_extracti128_si256(a,1),simde_mm256_extracti128_si256(b,1)),simde_x_mm_deinterleaveodd_epi16(simde_mm256_extracti128_si256(a,1),simde_mm256_extracti128_si256(b,1)));
 
   return simde__m256i_from_private(r_);
 #endif
 }
 #if defined(SIMDE_X86_AVX2_ENABLE_NATIVE_ALIASES)
-  #undef _mm256_hadd_epi16
+  #undef _mm256_hadd_epi16hadd
```

Comment:
?

Reply:
I will fix this, this has happened because I was trying to search the function in the file and didn't press "Ctrl+F" properly, and I didn't notice this.

Reply:
Np.  I have similar issues because I start typing in the wrong window.  You can usually catch stuff like this with a quick `git diff` before the committing, but stuff sometimes slips through.  It's nice to have a code review process to catch stuff like this.